app/components/list-features-deprecations.js

```diff
@@ -1,29 +1,34 @@
 import Component from '@glimmer/component';
 import { computed } from '@ember/object';
 import { compare } from 'compare-versions';
+import { filterBy } from '@ember/object/computed';
 
 export default class ListFeaturesDeprecationsComponent extends Component {
-  @computed('args.{datum,fromVersion,toVersion}')
-  get model() {
-    return this.args.datum.filter((model) => {
+  @computed('args.{allChangeLogs,fromVersion,toVersion}')
+  get relevantChangeLogs() {
+    return this.args.allChangeLogs.filter((changeLog) => {
```

Comment:
Just in case `@allChangeLogs` is `undefined`, can we add a guard so that `this.flattenedChangeLogs`, `this.deprecations`, and `this.features` can still be well-defined (they would be an empty array)?

```suggestion
    const { allChangeLogs } = this.args;

    if (!allChangeLogs) {
      return [];
    }

    return allChangeLogs.filter((changeLog) => {
```

Reply:
Good catch!